dev_setup/README.md

```diff
@@ -1,23 +1,104 @@
 # Setup Dev 
- + install docker
- + install docker compose
- + install npm
- + enable yarn
- + clone netbox repo
- + clone netbox_topology_views repo
- + create venv: `/usr/bin/python3 -m venv venv`
- + activate venv: `source venv/bin/activate`
- + install deps: `pip3 install -r requirements.txt`
- + install build tools: `python3 -m pip install --upgrade build setuptools wheel`
 
+We will assume that you work in the folder `~/code/netbox_plugin/`, change this if you use another folder for your projects/code.
+
+## Create the base folder
+```
+mkdir ~/code/netbox_plugin/
+cd ~/code/netbox_plugin/
+```
+
+## Install requirements
+### Python + deps
+You will need the following packages installed:
++ python3-venv
++ python3
+
+Install them with your OS package manager (apt, yum, ...)
+
+### Docker
+This depends on your OS or how you want to install docker.  
+See https://docs.docker.com/engine/install/ to find the install steps.
+
+### Docker compose
```

Comment:
Since Docker is optional this should be explained here and in the section "Start docker compose" as well.

Reply:
i added some changes to this in the latest commit, can you verify if its clear to you?

Reply:
Looks good so far. Just found a small typo.

One last thing: I would generally refrain from writing "python3" inside a venv, as it may lead to using the wrong python version. Instead, I would always write "python". This is always a link to the version with which the venv was created. If you have created the venv with Python3.x, "python3" is of course also correct. In the case of NetBox, this is therefore always correct. It is just a general point.

Reply:
I understand your point regarding python3 vs just python but in this case i will leave it at 'python3'. This is also in line with the docs from netbox itself.